JSDemos/Demos/Toolbar/Multiline/React/App.js

```diff
@@ -0,0 +1,323 @@
+            </Item>
+
+            <Item location="before" locateInMenu="auto">
+              <DropDownButton
+                width="100%"
+                displayExpr="text"
+                keyExpr="size"
+                useSelectMode={true}
+                items={fontSizes}
+                selectedItemKey={fontSize}
+                itemRender={renderFontSize}
+                onSelectionChanged={onFontSizeChange}
+              ></DropDownButton>
+            </Item>
+
+            <Item location="before" locateInMenu="auto">
+              <DropDownButton
+                width="100%"
+                icon="indent"
+                displayExpr="text"
+                keyExpr="lineHeight"
+                useSelectMode={true}
+                items={lineHeights}
+                selectedItemKey={lineHeight}
+                onSelectionChanged={onLineHeightChange}
+              ></DropDownButton>
+            </Item>
+
+            <Item location="before" locateInMenu="auto">
+              <SelectBox
+                placeholder="Font"
+                displayExpr="text"
+                dataSource={fontFamilies}
+              ></SelectBox>
+            </Item>
+
+            <Item
+              cssClass="toolbar-separator-container"
+              location="before"
+              locateInMenu="auto"
+              menuItemRender={renderMenuSeparator}
+            >
+              <div className="toolbar-separator"></div>
+            </Item>
+
+            <Item location="before">
+              <ButtonGroup
+                displayExpr="text"
+                keyExpr="style"
+                stylingMode="outlined"
+                selectionMode="multiple"
+                items={fontStyles}
+                onItemClick={onItemClick}
+              ></ButtonGroup>
+            </Item>
+
+            <Item location="before">
+              <div className="toolbar-separator"></div>
+            </Item>
+
+            <Item
+              cssClass="dx-toolbar-hidden-button-group"
+              location="before"
+              locateInMenu="auto"
+              render={renderTextAlign}
+              menuItemRender={renderTextAlignMenu}
+            ></Item>
+
+            <Item location="before" displayExpr="text">
+              <ButtonGroup
+                keyExpr="alignment"
+                stylingMode="outlined"
+                items={listTypes}
+                onItemClick={onItemClick}
+              ></ButtonGroup>
+            </Item>
+
+            <Item
+              cssClass="toolbar-separator-container"
+              location="before"
+              locateInMenu="auto"
+              menuItemRender={renderMenuSeparator}
+            >
+              <div className="toolbar-separator"></div>
+            </Item>
+
+            <Item location="before" locateInMenu="auto">
+              <DateBox
+                width={200}
+                type="date"
+                value={new Date(2022, 9, 7)}
```

Comment:
I'm not sure, but maybe we should use useMemo for this date?
Because, as far as I know, the new Date will be invoked on each render here

Reply:
We decided use a simple const for it. https://github.com/DevExpress/devextreme-demos/pull/2004/commits/b97c28826b4feb1b12e941e53837224b4c3838fb